pkg/rbac/sharedData.go

```diff
@@ -130,10 +214,11 @@ func (shared *SharedData) GetClusterScopedResources(cache *Cache, ctx context.Co
 		return shared.csErr
 	}
 
-	rows, queryerr := cache.pool.Query(ctx, query)
-	if queryerr != nil {
-		klog.Errorf("Error resolving query [%s]. Error: [%+v]", query, queryerr.Error())
-		shared.csErr = queryerr
+	// var queryerr error
+	rows, err := cache.pool.Query(ctx, query)
+	if err != nil {
+		klog.Errorf("Error resolving query [%s]. Error: [%+v]", query, err.Error())
+		shared.csErr = err
```

Comment:
Delete comment and update log message.
```suggestion
	rows, err := cache.pool.Query(ctx, query)
	if err != nil {
		klog.Errorf("Error resolving cluster scoped resources. Query [%s]. Error: [%+v]", query, err.Error())
		shared.csErr = err
```